### DISCUSSION TITLE
Pip install open3d not working in MacBook m1 : ERROR: Could not find a version that satisfies the requirement open3d (from versions: none) ERROR: No matching distribution found for open3d

### DISCUSSION THREAD

**Discussion by sidjr03** [isl-org/Open3D#6284] [Q&A]
<img width="562" alt="Screenshot 2023-07-26 at 4 17 34 PM" src="https://github.com/isl-org/Open3D/assets/139303516/bfb4bef7-fba8-4823-98a1-a2b91eeb003c">

**Comment by errissa** [comment]
What version of Python are you using?

**Comment by stockdam** [comment]
Open3D only runs on several versions of Python:

http://www.open3d.org/docs/0.11.0/getting_started.html

Best to use something like conda to create an environment where you can control the version of Python used.

I have it running successfully with JupyterLab but my notes say that i had to install libomp.

Note that Open3D visualization windows do not close correctly when using an M1 Mac (hopefully there is a fix for this or way round it).
